reclaimer/meta/wrappers/src/byteswapping_ext.c

```diff
@@ -0,0 +1,144 @@
+#include "..\..\..\src\shared.h"
+
+
+static void byteswap_eight_byte_array(
```

Comment:
Seeing the use case these really should be `static inline`
Avoid the jumps!

Reply:
I guess.... It's not a huge overhead, but considering what the functions are, they're practically screaming to be inlined.